src/openeo_gfmap/utils/catalogue.py

```diff
@@ -107,19 +133,20 @@ def _check_cdse_catalogue(
     return len(grd_tiles) > 0
 
 
-def s1_area_per_orbitstate(
+def s1_area_per_orbitstate_vvvh(
```

Comment:
what actually fails when using a different polarization? 

Could there be future GF MAP users who are interested in different polarization states?